app/components/Dashboard/Cards/WhatDifference.tsx

```diff
@@ -27,43 +35,43 @@ const Highlight: React.FC<React.PropsWithChildren> = ({ children }) => (
   <span className="text-green-500 font-semibold">{children}</span>
 )
 
-const Cards = () => (
+const Cards: React.FC<CardsProps> = ({ household }) => (
   <div className="flex flex-wrap gap-6">
-    <Card title="Money saved" figure="£30,000">
-      <p>On housing costs over 10 years, compared to conventional ownership</p>
+    <Card title="Money saved" figure={`£${Math.round(household.socialValue.moneySaved).toLocaleString()}`}>
+      <p>If Fairhold Land Purchase, on housing costs over 10 years compared to conventional ownership</p>
     </Card>
     <Card title="Community wealth">
       <p>
         Every 10 years the house would contribute{" "}
-        <Highlight>£120,000</Highlight> to community infrastructure and services
+        <Highlight>£{Math.round(household.socialValue.communityWealthDecade).toLocaleString()}</Highlight> to community infrastructure and services
```

Comment:
nit: It might be more readable to encode these as variables first rather than doing it all in-line.